npm/npm_import.bzl

```diff
@@ -304,17 +305,28 @@ def npm_translate_lock(
 
             See https://github.com/bazelbuild/bazel/issues/8106
 
-        verify_patches: Path to a workspace folder containing all patches used in the `patches` attribute. Will
-            fail if any patches are missing. Does not recurse into subfolders.
+        verify_patches: Label to a patch list file.
+
+            Use this in together with the `list_patches` macro to guarantee that all patches in a patch folder
+            are included in the `patches` attribute.
 
             For example:
 
             ```
-            verify_patches = "path/to/patches",
+            verify_patches = "//patches:patches",
```

Comment:
in this case it would be `"//patches:patches.list"  to match the `out` of the `list_patches` in the example below